notebooks/individual-dashboards/erroranalysis-dashboard/erroranalysis-dashboard-regression-superconductor.ipynb

```diff
@@ -86,7 +87,13 @@
    "source": [
     "outdirname = 'superconduct'\n",
     "zipfilename = outdirname + '.zip'\n",
-    "urlretrieve('https://archive.ics.uci.edu/static/public/464/superconductivty+data.zip', zipfilename)\n",
+    "url = 'https://archive.ics.uci.edu/static/public/464/superconductivty+data.zip'\n",
+    "# temporary workaround for UCI repository until website SSL certificate\n",
+    "# is renewed with requests instead of urlretrieve\n",
+    "# urlretrieve(url, zipfilename)\n",
+    "content = requests.get(url, verify=False).content\n",
+    "with open(zipfilename, mode='wb') as localfile:\n",
+    "    localfile.write(content)\n",
```

Comment:
Where is this `localfile` variable or associated file used?

Reply:
line 96, localfile.write(content)
it's just a temporary variable from open("my zip file", "wb")